src/client/inputhandler.h

```diff
@@ -285,6 +288,33 @@ class RealInputHandler : public InputHandler
 	{
 		return m_receiver->WasKeyReleased(keycache.key[k]) || joystick.wasKeyReleased(k);
 	}
+	virtual float getMovementSpeed() {
```

Comment:
Style: Bracket placement should be on a separate line. Same for the other newly added functions.
https://dev.minetest.net/Code_style_guidelines

Reply:
https://github.com/minetest/minetest/pull/11262/commits/537aca607c5f2164834c7b797c8afedbe0533f92